lib/howl/application.moon

```diff
@@ -316,12 +324,29 @@ class Application extends PropertyObject
 
       howl.janitor.start!
 
+    -- load files from command line
+    loaded_buffers = {}
     for path in *files
       file = File path
-      buffer = @new_buffer mode.for_file file
-      buffer.file = file
-      signal.emit 'file-opened', :file, :buffer
+      buffer = @_buffer_for_file file
+      unless buffer
+        buffer = @new_buffer mode.for_file file
+        buffer.file = file
+        signal.emit 'file-opened', :file, :buffer
```

Comment:
Similar code in `open_file` above is pcall protected but here it is not - should it be?